operator/config/crd/patches/cainjection_in_metrics_keptnmetrics.yaml

```diff
@@ -0,0 +1,7 @@
+# The following patch adds a directive for certmanager to inject CA into the CRD
```

Comment:
i think this file needs to be removed since we don't use cert-manager anymore. please make sure that it's also picked up by .gitignore

Reply:
the file is actually covered by gitignore already